game-core/src/main/java/games/strategy/engine/pbem/ForumPosterComponent.java

```diff
@@ -90,10 +90,7 @@ public ForumPosterComponent layoutComponents(
       final IAbstractForumPosterDelegate forumPosterDelegate,
       final TripleAFrame frame,
       final boolean hasPosted,
-      final boolean allowIncludeTerritorySummary,
-      final boolean allowIncludeProductionSummary,
-      final boolean allowDiceBattleDetails,
-      final boolean allowDiceStatistics) {
+      final boolean includeDetailsAndSummary) {
```

Comment:
This is a consolidation of the 4 parameters to one, each child class was uniform in returning a true or false, so we only need one value here.